src/main/java/kitchenpos/menus/tobe/domain/menuproduct/TobeMenuProductPrice.java

```diff
@@ -0,0 +1,56 @@
+package kitchenpos.menus.tobe.domain.menuproduct;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import java.math.BigDecimal;
+import java.util.Objects;
+
+@Embeddable
+public class TobeMenuProductPrice {
+    @Column(name = "price", nullable = false)
+    private BigDecimal price;
+
+    protected TobeMenuProductPrice() {
+    }
+
+    public TobeMenuProductPrice(BigDecimal price) {
+        if (Objects.isNull(price)) {
+            throw new IllegalArgumentException("메뉴 상품의 가격은 필수로 입력 해야 합니다.");
+        }
+
+        if (price.compareTo(BigDecimal.ZERO) < 0) {
+            throw new IllegalArgumentException("메뉴 상품의 가격은 0원 이상이어야 합니다. price : " + price);
+        }
+
+        this.price = price;
+    }
+
+    public static TobeMenuProductPrice multiply(final BigDecimal price, final long quantity) {
+        return new TobeMenuProductPrice(price.multiply(BigDecimal.valueOf(quantity)));
+    }
+
+    public static TobeMenuProductPrice zero() {
+        return new TobeMenuProductPrice(BigDecimal.ZERO);
+    }
+
+    public TobeMenuProductPrice sum(final TobeMenuProductPrice price) {
+        return new TobeMenuProductPrice(this.price.add(price.getPrice()));
+    }
```

Comment:
클래스 내부에서는 필드에 접근 바로 가능해요. getter를 사용하지 않아도 괜찮습니다. 

```suggestion
    public TobeMenuProductPrice sum(final TobeMenuProductPrice price) {
        return new TobeMenuProductPrice(this.price.add(price.price));
    }
```